firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -63,22 +62,6 @@ internal constructor(
     }
   }
 
-  /** Register the [subscriber]. This must be called for every dependency. */
-  fun register(subscriber: SessionSubscriber) {
-    if (!settings.sessionsEnabled) {
-      Log.d(TAG, "Sessions SDK disabled. Ignoring dependency registration.")
-      return
-    }
-
-    FirebaseSessionsDependencies.register(subscriber)
-
-    Log.d(
-      TAG,
-      "Registering Sessions SDK subscriber with name: ${subscriber.sessionSubscriberName}, " +
-        "data collection enabled: ${subscriber.isDataCollectionEnabled}"
-    )
-  }
-
   /** Calculate whether we should sample events using [sessionSettings] data. */
```

Comment:
This comment is left over from an earlier change. Can you delete it since you're already working in this file?